style.css

```diff
@@ -1,424 +1,547 @@
-:root {
-  --bg-purple: #6070ff;
-  --bg-dark-blue: #172b4d;
-  --bg-lavender: #ebebff;
-  --bg-slate-grey: #505f79;
-  --text-dark-blue: #344563;
-  --font-poppins: "Poppins", sans-serif;
-}
-
-*,
-*::after,
-*::before {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-}
+@import url("https://fonts.googleapis.com/css2?family=Poppins&family=Ubuntu:wght@300&display=swap");
 
-*:not(i) {
-  font-family: var(--font-poppins);
+* {
+  font-family: "Poppins", sans-serif;
 }
 
-.after-bdy {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  background-color: rgb(229, 226, 226);
+body {
+  margin: 0;
+  overflow-x: hidden;
 }
 
-h1 {
-  color: var(--text-dark-blue);
-  font-size: 40px;
-  font-weight: 700;
-  line-height: 52px;
+.container {
+  padding-left: 5%;
+  padding-right: 5%;
 }
 
-h3 {
-  color: var(--bg-dark-blue);
-  font-size: 32px;
-  font-weight: 700;
-  line-height: 44px;
+.fixed-top {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: 1030;
 }
 
-h6 {
-  color: var(--bg-purple);
-  font-weight: 500;
-  font-size: 16px;
-  line-height: 24px;
+.navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  padding-top: 5px;
+  padding-bottom: 5px;
 }
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓

Reply:
#### UPDATES TO ALL CSS RULES WITH A DISPLAY PROPERTY OF FLEX

- [ ] Hey, as requested by the previous reviewer on the issue of [repeated CSS rule](https://github.com/azeezumoru/microverse-portfolio/pull/4#discussion_r1116438040), he has stated perfectly the reasons why we should keep our codebase DRY. Also, according to your response [here](https://github.com/azeezumoru/microverse-portfolio/pull/4#issuecomment-1443434631), it seems you don't understand his suggestions well enough and you presume he is blocking your progress. I'd try to explain what you need to do to help resolve any uncertainty you're facing right now. please, kindly pay close attention below:

##### BELOW ARE IMAGES OF THE LINES OF CODE IN YOUR CSS FILE THAT HAS A REPETITIVE USE OF THE DISPLAY PROPERTY OF FLEX


![Screenshot from 2023-02-24 11-48-51](https://user-images.githubusercontent.com/59397159/221162225-e8c4738d-4cbf-400d-b672-b9d87c9f1e41.png)
![Screenshot from 2023-02-24 11-48-29](https://user-images.githubusercontent.com/59397159/221162241-96177c91-e326-4653-a71d-b48c83cb314d.png)
![Screenshot from 2023-02-24 11-47-43](https://user-images.githubusercontent.com/59397159/221162260-a35c183a-7124-4cf7-b077-d4efc7e7322f.png)
![Screenshot from 2023-02-24 11-47-21](https://user-images.githubusercontent.com/59397159/221162275-b39d6467-d7bc-4e21-9e27-48b4963ac282.png)
![Screenshot from 2023-02-24 11-46-56](https://user-images.githubusercontent.com/59397159/221162281-c9f1febd-2e56-4702-8bda-33ea0fd58a90.png)
![Screenshot from 2023-02-24 11-46-32](https://user-images.githubusercontent.com/59397159/221162291-e3a2d288-9c39-4265-8adf-370025e9cb37.png)
![Screenshot from 2023-02-24 11-46-09](https://user-images.githubusercontent.com/59397159/221162296-d1d5dd52-35b4-49b0-8a7f-e62d29610f3f.png)
![Screenshot from 2023-02-24 11-45-50](https://user-images.githubusercontent.com/59397159/221162299-f4f79232-efdf-4fae-8148-9f08c9c27a1b.png)
![Screenshot from 2023-02-24 11-45-36](https://user-images.githubusercontent.com/59397159/221162306-8b2f001b-bdce-4dc6-a87b-dd6373e54fd6.png)
![Screenshot from 2023-02-24 11-45-14](https://user-images.githubusercontent.com/59397159/221162315-496c81d0-47c1-4c6c-a6a6-24d2e30a6348.png)
![Screenshot from 2023-02-24 11-44-46](https://user-images.githubusercontent.com/59397159/221162329-9ea58e1a-6e7e-4994-89ba-15ec1752f50b.png)
![Screenshot from 2023-02-24 11-44-24](https://user-images.githubusercontent.com/59397159/221162335-b26e4634-7c10-4a00-87e8-bfdc2e76b05b.png)
![Screenshot from 2023-02-24 11-43-50](https://user-images.githubusercontent.com/59397159/221162340-e1158ac2-9bd6-4e18-9113-e7c8736a5681.png)
![Screenshot from 2023-02-24 11-43-25](https://user-images.githubusercontent.com/59397159/221162346-2c5a0d4c-7d86-40a0-bc1d-1c9f13402e28.png)
![Screenshot from 2023-02-24 11-43-12](https://user-images.githubusercontent.com/59397159/221162356-d4f624c7-7e36-4f74-bbe6-f7e8009b8916.png)


### WHAT TO DO 

- [ ] As the previous reviewer suggested, create a class like say `flex` 
```
.flex {
   display: flex;
}
```
- Add the class `flex` to all these classes where you've used the display property of flex. Here's an example on the navbar:

`<header class="navbar container fixed-top">` Since the `navbar` has a display property of flex, you can update it to this `<header class="flex navbar container fixed-top">` then remove the display: flex; from the `navbar` class in the CSS rule like this below:


```
.navbar {
  display: flex; /* remove this line totally. Do the same to every part of your CSS where you've used display: flex; */
  flex-direction: row;
  align-items: center;
  justify-content: space-between;
  background-color: #fff;
  padding-top: 5px;
  padding-bottom: 5px;
}
```